aten/src/ATen/native/transformers/sdp_utils_cpp.h

```diff
@@ -503,16 +503,9 @@ inline bool check_last_dim_stride_equals_1_dense(sdp_params const& params, bool
   if (ignore_singleton_dim){
     qkv_strides_equal_1 = qkv_strides_equal_1 || params.query.sym_size(-1) == 1;
   }
-  bool mask_stride_equal_1 = params.attn_mask.has_value()
-      ? params.attn_mask.value().sym_stride(-1) == 1
-      : true;
-  if (!(qkv_strides_equal_1 && mask_stride_equal_1)) {
+  if (!qkv_strides_equal_1) {
     if (debug) {
       std::ostringstream epilogue_message;
-      if (params.attn_mask.has_value()) {
-        epilogue_message << ", Attn_mask.stride(-1): "
-                         << params.attn_mask.value().sym_stride(-1);
-      }
       epilogue_message << " instead.";
```

Comment:
Remove this.

Reply:
Thanks for your comments. removed `epilogue_message`.